src/nxdoc/nuxeo-server/nuxeo-server-release-notes.md

```diff
@@ -11,1268 +11,318 @@ toc: true
-
-Three new methods have been added to the Audit service: `getLatestLogId`, `getLogEntriesAfter` and `queryLogs`
-
-<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXP-21661](https://jira.nuxeo.com/browse/NXP-21661).
-
-
-
-#### Audit Storage SPI {{since '9.3'}}
-
-A new interface `AuditStorage` has been added to define a storage back-end abstraction, so as to be able to store audit data in several places easily. A "storage" doesn't implement all the methods of the Audit service, but enough for storing audit entries and scroll them.
-
-<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXP-23291](https://jira.nuxeo.com/browse/NXP-23291).
-
-#### Restore From Storage {{since '9.3'}}
-
-* A new extension point is available to register an audit storage:
-
-```xml
-<extension target="org.nuxeo.ecm.platform.audit.service.NXAuditEventsService" point="storage">
-<storage id="defaultAuditStorage" class="org.nuxeo.ecm.platform.audit.service.DefaultAuditBackend"/>
-</extension>
-```
-
-* A new automation operation `Audit.Restore` is available to be able to restore an audit backend from an audit storage.
-
-<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXP-23292](https://jira.nuxeo.com/browse/NXP-23292).
-
-### CMIS
-
-#### Relax Mode {{since '9.1'}}
-
-Some constraints of CMIS can be bypassed. To do so use the runtime property `nuxeo.dontFollowCmisSpec=true`.
-This way, for instance, multiple "contains" can be used in the CMISQL query while the standard forbids it.
-
-<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXP-19858](https://jira.nuxeo.com/browse/NXP-19858).
-
-#### Change Token  {{since '9.1'}}
-
-Change Token [specified](http://docs.oasis-open.org/cmis/CMIS/v1.1/cs01/CMIS-v1.1-cs01.html#x1-1610003) in CMIS 1.1 as optional has been implemented.
-
-<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXP-20665](https://jira.nuxeo.com/browse/NXP-20665).
-
-#### Proxies Are Visible by Default  {{since '9.1'}}
-
-`org.nuxeo.cmis.proxies=true` is now the default.
-
-<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXP-21828](https://jira.nuxeo.com/browse/NXP-21828).
-
-#### maxItems Parameter in CMIS Query Is Now Supported {{since '9.2'}}
-
-When using the ES implementation of CMIS bridge, using the parameter `maxItems` allows you to get a certain amount of documents without loading the entire result set at once. Implementation over Core in case of DBS has also been improved consequently.
+#### Startswith Operator for Elasticsarch Audit Backend {{since '10.1'}}
+Startswith operator implementation has been completed in Elasticsearch audit backend implementation.
+Ex: {{ .addAndPredicate(new Predicate(new Reference("docPath"), Operator.STARTSWITH, Literals.toLiteral(input.getPath().toString()))) }}
```

Comment:
Code could be a bit better here.
Following introduction of `AuditBackend#queryLogs(AuditQueryBuilder)`, we now have an eaasy way to query audit. We introduce in 10.1 the `STARTSWITH` operator, we could use it as below:
```
auditBackend.queryLogs(new AuditQueryBuilder().predicates( //
                Predicates.eq(LOG_EVENT_ID, "SOMETHING"),
                Predicates.startsWith(LOG_DOC_PATH, "/myFolder")));
```

Reply:
Updated